extensions/wikia/SpecialDiscussionsLog/templates/SpecialDiscussionsLog_userLog.mustache

```diff
@@ -1,15 +1,18 @@
-{{#hasUserError}}
-	<p>{{userErrorMessage}}</p>
-{{/hasUserError}}
-{{^hasUserError}}
+{{#userErrorMessage}}
+	<div class="log-error-message">{{userErrorMessage}}</div>
+{{/userErrorMessage}}
+{{^userErrorMessage}}
 	{{#hasNoUserLogRecords}}
-		<p>{{noUserLogRecordsMessage}}</p>
+		<div class="log-error-message">{{noUserLogRecordsMessage}}</div>
 	{{/hasNoUserLogRecords}}
 	{{^hasNoUserLogRecords}}
-		<p>{{logTableCaption}}</p>
+		<div class="log-table-caption">
```

Comment:
There's a paragraph inside a div here. Maybe we could get rid of one since it seems the div has nothing else except stated paragraph? :neckbeard: 

Also, there is an html tag called "caption" (https://developer.mozilla.org/pl/docs/Web/HTML/Element/caption), which seems to serve the exact same purpose as this div. The only thing here is that it should be placed inside the table.